mobile/library/cc/engine_builder.h

```diff
@@ -167,6 +167,7 @@ class EngineBuilder {
 #if defined(__APPLE__)
   // TODO(abeyad): once stable, consider setting the default to true.
   bool respect_system_proxy_settings_ = false;
+  int ios_network_service_type_ = 0;
```

Comment:
I think using `absl::optional` is probably better than using 0 because 0 means `NSURLNetworkServiceTypeDefault`: https://developer.apple.com/documentation/foundation/nsurlrequestnetworkservicetype/nsurlnetworkservicetypedefault

Reply:
Yeah, I thought about that, but I think b/c it's the default, in the Cronet implementation, 0 is used to abort early: https://source.chromium.org/chromium/chromium/src/+/main:net/socket/udp_socket_posix.cc;l=1084-1086;drc=7f04a8e033c23dede6beae129cd212e6d4473d72

Reply:
(i.e. there's no need to set the sock opt)